01_fib/ovlords/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func fib(n int) {
+
+	if n > 0 {
+
+		currentNumber, previousNumber, scratchNumber := 1, 0, 0
+
+		for lines := 1; lines <= n; lines++ {
+			fmt.Fprintln(out, currentNumber)
+
+			scratchNumber, previousNumber = previousNumber, currentNumber
+			currentNumber += scratchNumber
```

Comment:
These two lines can be combined into 1 by taking advantage of Go's multiple assignment form:
```
a, b = b, a+b
```
When done this way, you don't need the scratch variable.